target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4028-wpj428.dts

```diff
@@ -32,6 +32,10 @@
 
 		mdio@90000 {
 			status = "okay";
+			pinctrl-0 = <&mdio_pins>;
+			pinctrl-names = "default";
+			reset-gpios = <&tlmm 59 GPIO_ACTIVE_LOW>;
```

Comment:
Is GPIO_ACTIVE_LOW correct? The [Compex Lede repository](https://github.com/compex-systems/lede/blob/openwrt-18.06/target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4028-wpj428.dts) contains:
`    phy-reset-gpio = <&tlmm 59 0>;`
Which I think means GPIO_ACTIVE_HIGH.